src/components/Button/Button.scss

```diff
@@ -24,5 +24,6 @@
   &--active {
     color: $grey;
     background-color: $yellow;
+    cursor: pointer;
```

Comment:
Fine to leave this for now, but I would avoid any styling changes for the time being. Going to be adding in Tailwind CSS at some point over the next day or two so it'll all need changing then anyway